src/main/java/com/micetr0/model/Measure.java

```diff
@@ -6,9 +6,39 @@
  */
 
 package com.micetr0.model;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.lang.Exception;
+import java.util.List;
 
 public class Measure {
+    private Double sizeLimit;
+    private List<Note> notes = new ArrayList<Note>();
+    private int currIndex;
 
-    //TODO: Implement
+
+    public Double getSizeLimit() {
+        return sizeLimit;
+    }
+
+    public List<Note> getNotes() {
+        return notes;
+    }
+
+    public void setSizeLimit(Double sizeLimit) {
+        this.sizeLimit = sizeLimit;
+    }
+
+    public void setNotes(List<Note> notes) {
```

Comment:
dumb, get rid of. blame rick

Reply:
Deleted everything in measure class. Starting over